modules/virt-enabling-disabling-sidecar-injection.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-managing-sidecar-injection.adoc
+
+[id="virt-enabling-disabling-sidecar-injection_{context}"]
```

Comment:
So the side car injection we are talking about here is specific to istio, other components may have other mechanisms for side car injection. So this is not a generic method for enabling/disabling side car injection. I would definitely specify that this is istio specific.